core/wrapper/socestimator_test.go

```diff
@@ -12,7 +12,7 @@ import (
 func TestRemainingChargeDuration(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	vehicle := mock.NewMockVehicle(ctrl)
-	vehicle.EXPECT().Capacity().Return(int64(10))
```

Comment:
Wieso sollte sich hier etwas ändern? Das macht für mich keinen Sinn. Funktioniert dieser Test?

Reply:
Weil sonst die nachfolgenden Tests fehlschlagen, da diese von 10 kWh virtualBatCap ausgehen.

Reply:
Mhhhm. Trotzdem ist das eigentlich falsch da ein echtes Auto hier etwas anderes zurück geben würde. Eigentlich sollte dann eher der Effizienzparameter in die erwarteten Ergebnisse mit eingehen oder hier wenigstens ein Kommentar rein wie die Zusammenhänge sind. Das versteht sonst später kein Mensch mehr (so wie ich jetzt schon nicht).

Reply:
Man könnte stattdessen auch alle weiteren Tests neu berechnen und modifizieren, richtig.
Am Testresultat ändert das aber nichts.
Nur alle anderen Werte wären wesentlich unleserlicher und zudem alle anders als bei dieser minimalinvasiven Änderung der vom Nutzer angegebenen Batteriegröße um wieder auf 10 kWh zu kommen.

Reply:
Ich möchte das egtl. nicht diskutieren. Die andere Iption ist ein Kommentar wie geschrieben.